src/services/pool/calculator/index.ts

```diff
@@ -81,7 +84,7 @@ export default class Calculator {
       amounts.send.forEach((amount, amountIndex) => {
         const greaterThanBalance =
           Number(amount) >
-          this.allTokens[this.tokenOf(type, amountIndex)].balance;
+          Number(this.allTokens[this.tokenOf(type, amountIndex)].balance);
```

Comment:
This coercion in JavaScript would work fine, but TypeScript doesn't like it... so gotta convert it to a number first before doing the math operation.